### PR TITLE
return sway config options

### PR DESCRIPTION
**Pull request by leviathanbeak**
Client reads Sway Configuration on sends it to the Server

### CODE REVIEW

**Comment by adlerjohn** [comment]
So these would be fields in the user's workspace or global `settings` file?

**Comment by leviathanbeak** [comment]
> So these would be fields in the user's workspace or global `settings` file?

they are read from global `settings.json`

either like this
```
"[sway]": {
    "format.align_fields": false
},
```

or like this
```
"sway.format.alignFields": false,
```

in LSP/forc we can give preference to workspace/manifest settings

File: client/src/extension.ts [hunk 1]
```diff
@@ -75,8 +75,51 @@ function getClientOptions(): LanguageClientOptions {
...
+		return swayOptionsBracket
+	} else {
+		return null
+	}
+}
+
+type SwayConfig = {
+	alignFields: boolean,
+	tabSize: number,
 }
```
Comment (alex-matthe) [inline client/src/extension.ts:125]: new line warning

**Review by alex-matthe — DISMISSED** [review]
👍

**Review by adlerjohn — DISMISSED** [review]
utACK, but if you can add a newline at the end of the file that would be great.

**Review by AlicanC — DISMISSED** [review]

**Comment by leviathanbeak** [comment]
ok, added the newline, please re-approve, @adlerjohn @AlicanC @alex-matthe

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**